suite-native/notifications/src/components/ToastNotification.tsx

```diff
@@ -46,54 +47,84 @@ const IconContainerStyle = prepareNativeStyle<{
     ),
 }));
 
-const ToastNotificationVariantToStyleMap = {
+const ToastTypeToContentMap = {
+    'copy-to-clipboard': {
```

Comment:
Why do we need all these types? Why not just have type `toast` and pass all these info like `variant`, `iconName`, `message` with every call?

It will be much more flexible and we won't need to predefine all of possible types. Just do something like `dispatch(addNotification({ type: 'toast', icon: 'copy', variant: 'default', message: 'Hello copied.' })`